package m_Chapter;

public class A_Exception {
	public static void main(String[] args) {
		
		try {
			int division = 10/0;
		}catch(Exception e) {
			System.out.println("0으로 나눌 수 없습니다.");
		}
	}
}
